src/allmydata/node.py

```diff
@@ -713,7 +713,7 @@ class Node(service.MultiService):
     """
     NODETYPE = "unknown NODETYPE"
     CERTFILE = "node.pem"
-    GENERATED_FILES = []
+    GENERATED_FILES = []  # type: ignore
```

Comment:
Sure. I'd observed the same, but gave the code the benefit of the doubt and left it around in case it was used through some mysterious usage I couldn't detect. Fixed in 189608e11.